documentation/querying-vespa.html

```diff
@@ -4,74 +4,79 @@
-    which dispatch queries to <a href="proton.html">proton</a> search nodes.
-    For better caching, queries have affinity to TLDs.
-  </li>
+<li>The search phase which does matching, ranking and grouping/aggregation. This phase involves dispatching the query out to content nodes</li>
+<li>The result processing phase which involves content fetching and snippeting of the top global hits found in the previous search phase</li>
 </ol>
-<p>
-<img src="img/reference/flow/query1.png" />
-</p><p>
-At this point the query enters one or more <a href="reference/services-content.html">content clusters</a>:
-</p>
-<ol start="4">
+<p>The above is an simplification as if the search request also specifies <a href="grouping.html">result grouping</a> 
+the search phase might involve multiple phases or round trips between the container and the content nodes</p>
+
+
+<h2 id="life-of-a-query">Life of a Query </h2>
+<h3 id="first-phase">The search phase</h3>
+<ol>
   <li>
-    The query arrives at each content node's mid-level dispatcher (MLD), which is the same software as the TLD.
-    <a href="content/data-placement.html">Grouped clusters</a> are an exception to this,
-    where only a subset of content nodes are queried. The MLD forwards the query to <em>vespa-proton</em>.
+    A query is sent from a front-end application to a container node over HTTP(s) using GET or POST
+    using the <a href="search-api.html">search API</a> or in any custom request format handled by a 
+    custom <a href="jdisc/developing-request-handlers.html">request handler</a> which translates the custom request format to native Vespa APIs.
   </li><li>
-    <em>vespa-proton</em> searches its index and returns its results to the MLD.
+    Query pre-processing, like <a href="linguistics.html">linguistic processing</a> and <a href="query-rewriting.html">query rewriting</a>,
+    is done in <a href="chained-components.html">configured search chains</a>
   </li><li>
-    When all MLDs have responded to the TLD,
-    it merges the partial results and responds to the search container.
+    The query is sent from the container to <a href="search-definitions.html#searching-multiple-document-types">selected</a> content clusters -
+    see <a href="federation.html">federation</a> for more details. The illustration above only
+   contains one content cluster but multiple is fully supported and allows scaling <a href="search-definitions.html">document types</a> differently. 
+   For example a <em>tweet</em> document type could be indexed in a separate content cluster then <em>user</em> document type which allow for independent scaling. 
   </li><li>
-    The search container performs processing of the results in search chains
-    making further queries if needed before it returns the results to the querying entity.
+    Content clusters have top-level dispatchers (TLDs) which dispatches the query to <a href="content/data-placement.html">content groups</a> within the content cluster.
```

Comment:
as dispatch goes away shortly I would just remove this

Reply:
we will have to remove dispatch from illustrations, too, can do in later PR

Reply:
I think we can leave this until we know how rpc first phase search is going to be enabled 100%